pkg/scheduler/profile/profile.go

```diff
@@ -123,7 +123,7 @@ func (v *cfgValidator) validate(cfg config.KubeSchedulerProfile) error {
 		return fmt.Errorf("different queue sort plugins for profile %q: %q, first: %q", cfg.SchedulerName, queueSort, v.queueSort)
 	}
 	if !cmp.Equal(v.queueSortArgs, queueSortArgs) {
-		return fmt.Errorf("different queue sort plugin args for profile %q: %s", cfg.SchedulerName, queueSortArgs.Raw)
+		return fmt.Errorf("different queue sort plugin args for profile %q: %s", cfg.SchedulerName, queueSortArgs)
```

Comment:
printing `queueSortArgs` as a string probably won't output what you want

Reply:
I'll just remove the detail, as the given information should be enough to troubleshoot.